test/dlc_tests/container_tests/bin/testTFAddons

```diff
@@ -13,7 +13,7 @@ elif [[ $(python -c 'import tensorflow as tf; print(tf.__version__)') == 2.2.* ]
 else
   pip install -U --no-deps tensorflow_addons
 fi
-pip install typeguard
+pip install typeguard packaging
```

Comment:
`packaging` is not explicitly installed in the prod image. Please suggest if this should go in the dockerfile

Reply:
Looking through the repo, it looks like we have not been installing this in our TF images, instead relying on our test helpers to install it. I think leaving it out of the docker file is fine.

 For what it's worth, we have been installing it in some of our PT and HF images, so putting it in the image wouldn't be unique.
 
 That being said, since it hasn't been in our TF images and is used only for helping with tests, I'd say we leave it out of our docker file.